### PR TITLE
Include test for firewall customizations

### PR DESCRIPTION
**Pull request by mcattamoredhat**
This PR includes firewall customization in upgrade's blueprint for edge-commit.
A playbook task has been included to ensure that zones and IP addresses are configured in the firewall.

### CODE REVIEW

File: check-ostree.yaml [hunk 1]
```diff
@@ -899,6 +900,33 @@
...
+            fail_msg: "No firewall customizations found"
+            success_msg: "Firewall customizations added from blueprint"
+      always:
+        - set_fact:
+            total_counter: "{{ total_counter | int + 1 }}"
+      rescue:
+        - name: failed count + 1
+          set_fact:
+            failed_counter: "{{ failed_counter | int + 1 }}"
+      when: firewall_feature == "true"
```
Comment (henrywang) [inline check-ostree.yaml:928]: 1. The checking should be running on RHEL 8.8/9.2 and above, CS8/9, and Fedora, like https://github.com/virt-s1/rhel-edge/pull/1665/files#diff-e719b01c473575efb6bc5c7f6423d53ddbca736721550cd078e3c9b1a5b154e4R281

**Review by henrywang — CHANGES_REQUESTED** [review]
Great job @mcattamoredhat. Thanks for doing this.
I have a comment in this PR.

File: check-ostree.yaml (hunk 1 above)
Comment (henrywang) [reply check-ostree.yaml:928]: I saw you did this in ostree.sh script. But this ansible playbook will run on all distros.

Comment (mcattamoredhat) [reply check-ostree.yaml:960]: Variable firewall_feature: "false" should avoid this.

**Review by henrywang — APPROVED** [review]
Great job!

**Pull request merged**